modules/nvidia-gpu-admin-dashboard-using.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
+
```

Comment:
Here you may want to mention that you have to click on refresh the console, there's a message that pops up in top right pane of console after installing and enabling the console pugin.  Once you click on refresh, it will show the GPUs tab under Compute.

Reply:
Added "To view the changes, you need to refresh the console to see the GPUs tab under Compute.".